googletrans/constants.py

```diff
@@ -1,5 +1,72 @@
 DEFAULT_USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64)'
 
+DEFAULT_SERVICE_URLS = ('translate.google.ac','translate.google.ad','translate.google.ae',
```

Comment:
Wouldn't be better to have it generated with loops ?

```
FORMATS = [
    'translate.google.{}',
    'translate.google.co.{}',
]

LANGS = ['ac', 'ad', 'ae', ...]

DEFAULT_SERVICE_URLS = [f.format(lang) for lang in LANGS for f in FORMATS]
```

Or at least explain where this list comes from ?

Reply:
I'm not sure it's good idea to do so because the suffixes of domains are not language code, it's either TLD or ccTLD.